### PR TITLE
New Get Started section for Performance: Index

### PR DESCRIPTION
**Pull request by dgash**
New Get Started section for Performance: Index

### CODE REVIEW

**Comment by addyosmani** [comment]
Lgtm

**Comment by addyosmani** [comment]
@Meggin Is the plan to wait until the other docs Dave has authored are ready before trying to bring this content in (seemed like that was perhaps the case but just wanted to confirm).

File: src/content/en/fundamentals/instant-and-offline/get-started/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+* Users visit more often, stay longer, search more, and buy more frequently on sites that load quickly than on slower ones; a usage reduction of 0.5% (very significant at today's internet traffic rates) can occur even when the slowdown is less than half a second (source: [Google Research Blog](https://research.googleblog.com/2009/06/speed-matters.html))
+
+* Slow loading is detrimental for search engine optimization (SEO) because it can lower your site's ranking, resulting in fewer visits, reads, and conversions; Google recently implemented site speed as a ranking signal in its searches (source: [Google Webmaster Central Blog](https://webmasters.googleblog.com/2010/04/using-site-speed-in-web-search-ranking.html)).
+
+# The Goal
+
+Developers often talk about measurable concepts like Time To Interactivity (TTI) or First Meaningful Paint (FMP). These are attempts to quantify an important and overarching goal, which is simply that *pages should load as quickly as possible* on the first visit and, especially, on subsequent visits so that users can access both content and behavior at the earliest possible moment.
```
Comment (addyosmani) [inline src/content/en/fundamentals/instant-and-offline/get-started/index.md]: A beginner may not know about these metrics. I'd recommend we link out to docs on them like https://developers.google.com/web/updates/2017/06/user-centric-performance-metrics.

File: src/content/en/fundamentals/instant-and-offline/get-started/index.md [hunk 2]
```diff
@@ -16,39 +16,49 @@ better apps. Everyone benefits when we write pages that load and run faster, inc
...
+[SOASTA Google study report](https://soasta.com/blog/google-mobile-web-performance-study/)).
 
-* Users visit more often, stay longer, search more, and buy more frequently on sites that load 
-quickly than on slower ones; a usage reduction of 0.5% (very significant at today's internet traffic
- rates) can occur even when the slowdown is less than half a second (source: 
- [Google Research Blog](https://research.googleblog.com/2009/06/speed-matters.html)).
+- Users visit more often, stay longer, search more, and buy more frequently on sites that load 
+quickly than on slower ones; a usage reduction of 0.5% (very significant at today's internet traffic 
+rates) can occur even when the slowdown is less than half a second (source: 
+[Google Research Blog](https://research.googleblog.com/2009/06/speed-matters.html)).
```
Comment (addyosmani) [inline src/content/en/fundamentals/instant-and-offline/get-started/index.md]: Per my previous comment, can we please not reference any statistics that are older than 5 years? The 2009 stat here is still being used. Instead, please find a replacement stat from wpostats.com that works as well or we can alternatively drop this reference/note.

**Comment by petele** [comment]
Looks like the build is failing, once that's fixed, @addyosmani, please **_squash and merge_** when you're ready.

**Pull request merged**